src/_layouts/default.html

```diff
@@ -4,13 +4,9 @@
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
 
-    {%- capture title -%}
-      {% assign this_title = page.title | default: post.title %}
-      {% if this_title %}{{ this_title }} | {% endif %}{{ site.title }}
-    {%- endcapture %}
-    {% assign description = page.description | default: site.description %}
-
-    {% seo title=title description=description %}
+    {%- capture title -%} {% assign this_title = page.title | default: post.title %} {% if this_title %}{{ this_title }} | {%
+    endif %}{{ site.title }} {%- endcapture %} {% assign description = page.description | default: site.description %} {% seo
+    title=title description=description %}
```

Comment:
Was this change intentional? I feel like how it was before was easier to read

Reply:
This is automatically coming from the linter. I do agree it's easier to read before. Not sure if I can override it.